### PR TITLE
PR / Issue Review process update

### PR DESCRIPTION
**Pull request by bretg**
## Type of change

- [ ] Other

## Description of change

Updated process documentation as discussed in the Prebid Org meeting today.

### CODE REVIEW

File: pr_review.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
...
+We take PR review seriously. Please read https://medium.com/@mrjoelkemp/giving-better-code-reviews-16109e0fdd36#.xa8lc4i23 to understand how a PR review should be conducted. Be rational and strict in your review, make sure you understand exactly what the submitter's intent is. Anyone in the community can review a PR, but a Prebid Org member is also required. A Prebid Org member should take ownership of a PR and do the initial review.
+
+If the PR is for a standard bid adapter or a standard analytics adapter, just the one review is sufficient. The reviewer will check against [required conventions](http://prebid.org/dev-docs/bidder-adaptor.html#required-adapter-conventions) and may merge the PR after approving and confirming that the documentation PR against prebid.org is open and linked to the issue.
```
Comment (mkendall07) [inline pr_review.md]: just the one review [from a core team member] is sufficient

File: pr_review.md [hunk 2]
```diff
@@ -23,3 +27,20 @@ We take PR review seriously. Please read https://medium.com/@mrjoelkemp/giving-b
 - Verify that code re-use is being done properly and that changes introduced by a bidder don't impact other bidders.
 - If the adapter being submitted is an alias type, check with the bidder contact that is being aliased to make sure it's allowed.
 - If the adapter is triggering any user syncs make sure they are using the user sync module in the Prebid.js core.
+- Requests to the bidder should be in HTTPS
+- Responses from the bidder should be compressed
```
Comment (mkendall07) [inline pr_review.md]: (such as gzip, compress, deflate)

File: pr_review.md [hunk 3]
```diff
@@ -23,3 +27,20 @@ We take PR review seriously. Please read https://medium.com/@mrjoelkemp/giving-b
 - Verify that code re-use is being done properly and that changes introduced by a bidder don't impact other bidders.
 - If the adapter being submitted is an alias type, check with the bidder contact that is being aliased to make sure it's allowed.
 - If the adapter is triggering any user syncs make sure they are using the user sync module in the Prebid.js core.
+- Requests to the bidder should be in HTTPS
```
Comment (mkendall07) [inline pr_review.md]: should support HTTPS (doesn't have to be HTTPS 100% of the time)

**Comment by bretg** [comment]
Not sure the best way to rename a file...  copy and delete?  @mkendall07 ?

**Comment by GLStephen** [comment]
@bretg "git mv pr_review.md PR_REVIEW.md" will track the file change without Git losing its mind.

**Comment by mkendall07** [comment]
@bretg 
just waiting on the file rename and this can merge.

**Pull request merged**